src/winfile.h

```diff
@@ -1037,8 +1037,8 @@ Extern DWORD (CALLBACK *lpfnWNetFormatNetworkNameW)(
                     DWORD    dwFlags,
                     DWORD    dwAveCharPerLine
                     );
-Extern DWORD (CALLBACK *lpfnShareCreate)(HWND);
-Extern DWORD (CALLBACK *lpfnShareStop)(HWND);
+Extern DWORD (CALLBACK *lpfnShowShareFolderUI)(HWND, LPWSTR);
+Extern DWORD (CALLBACK *lpfnShareStop)(LPCWSTR, LPCWSTR, DWORD, HWND);

```

Comment:
Shouldn't this be removed? Along with NETWORK_ShareStop?

Reply:
Duplicate comment